packages/database/src/models/ChangeLog.ts

```
@@ -0,0 +1,93 @@
+  declare loggedAt: Date;
+  declare updatedByUserId: string;
+  declare recordId: string;
+  declare recordUpdate: boolean;
+  declare recordCreatedAt: Date;
+  declare recordUpdatedAt: Date;
+  declare recordDeletedAt: Date | null;
+  declare recordSyncTick: number;
+  declare recordData: string;
+
+  static initModel({ primaryKey, ...options }: InitOptions) {
+    super.init(
```

Biome (1.9.4)

[error] Using super in a static context can be confusing.

super refers to a parent class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)